### DISCUSSION TITLE
Memory Error on Windows

### DISCUSSION THREAD

**Discussion by addamit** [openai/whisper#235] [Q&A] [answered]
Hi - 
Keep getting MemoryError on a new windows laptop (16GB RAM). I tried medium and small model. 

Trying to troubleshoot. In my python file   I have:
```
import whisper
model = whisper.load_model("small")
```

The code keeps failing by spitting MemoryError:

```
Traceback (most recent call last):
  File "c:\Users\dec\workspace\py\whisper\whisper\example1.py", line 3, in <module>
    model = whisper.load_model("small")
  File "c:\Users\dec\workspace\py\whisper\whisper\whisper\__init__.py", line 97, in load_model
    checkpoint_file = _download(_MODELS[name], download_root, in_memory)
  File "c:\Users\dec\workspace\py\whisper\whisper\whisper\__init__.py", line 41, in _download
    model_bytes = open(download_target, "rb").read()
MemoryError
```

Not sure why this complains on memory error. I can see in my local model cache, both medium and small pt checkpoints available. 
Thanks in advance

**Comment by jongwook** [comment, marked as answer]
I'm not certain, but it's possible that other processes might be taking up most of the 16GB RAM and preventing the model to load. This is not very likely given the relatively small size of the model (462M), but can you try [increasing the page file size](https://www.thewindowsclub.com/increase-page-file-size-virtual-memory-windows)?

**Reply by addamit** [reply]
Thanks. Increasing the page file size helped. 
Now the example script runs except at the decode function call. 

```
result = whisper.decode(model, mel, options)
```

I keep getting error like so:
```
  File "C:\Users\dec\workspace\py\whisper\whisper_env\lib\site-packages\torch\nn\modules\conv.py", line 303, in _conv_forward
    return F.conv1d(input, weight, bias, self.stride,
RuntimeError: "slow_conv2d_cpu" not implemented for 'Half'
```

Not sure why.

**Reply by addamit** [reply]
Found from another post. 
options = whisper.DecodingOptions(fp16 = False), worked for me
